assets/javascripts/discourse/templates/components/global-filter-composer-container.hbs

```diff
@@ -0,0 +1,5 @@
+{{#if (and globalFilters this.siteSettings)}}
```

Comment:
You don't need to check for `this.siteSettings` here... global filters are coming from site setting, so this seems redundant.